src/state/selectors.js

```diff
@@ -40,57 +53,48 @@ export const getMultipleChoiceQuestions = createSelector(
   }, {})
 );
 
-export const getMultipleChoiceCounts = createSelector(
-  getMultipleChoiceQuestions,
-  getSelected,
-  getAggregations,
-  (mcQuestions, selected, aggregations) => Object.keys(mcQuestions).reduce((carry, key) => {
-    const question = mcQuestions[key];
-    const optionsWithCounts = question.options.map((option) => {
-      let optionCount = 0;
-
-      if (selected.emoji) {
-        optionCount = aggregations[selected.emoji][question.id][option.id];
-      } else if (aggregations) {
-        optionCount = Object.keys(aggregations)
-          .reduce((count, groupKey) => {
-            if (aggregations[groupKey][question.id][option.id]) {
-              return count + aggregations[groupKey][question.id][option.id];
-            }
-            return count;
-          }, 0);
-      }
-
-      return Object.assign({
-        count: optionCount
-      }, option);
-    });
-    return Object.assign(carry, {
-      [key]: optionsWithCounts
-    });
-  }, {})
-);
-
-export const getMultipleChoiceCountsList = createSelector(getMultipleChoiceCounts, objectToList);
-
 /**
- * Return an array of emoji multiple-choice answer objects with `.answer` and
+ * Return an array of emoji multiple-choice question objects with `.value` and
  * `.id` keys
  *
  * @param {Object} state The state object
- * @returns {Object[]} An array of `{ answer, id }` objects
+ * @returns {Object[]} An array of `{ value, id }` objects
  */
 export const getEmojiList = createSelector(
   getEmojiQuestion,
   emojiQuestion => (emojiQuestion && emojiQuestion.options) || []
 );
 
 /**
- * Return an array of emoji multiple-choice answer objects with counts for
+ * Return an array of topic (or focus area; "non-emoji") multiple-choice
+ * question objects with `.value` and `.id` keys
+ *
+ * @param {Object} state The state object
+ * @returns {Object[]} An array of `{ value, id }` objects
+ */
+export const getTopicList = createSelector(
+  getTopicQuestion,
+  topicQuestion => (topicQuestion && topicQuestion.options) || []
+);
+
+export const getSelectedEmoji = createSelector(
+  getEmojiList,
+  getSelected,
+  (emojiList, selected) => emojiList.find(emoji => emoji.id === selected.emoji)
+);
+
+export const getSelectedTopic = createSelector(
+  getTopicList,
+  getSelected,
+  (topicList, selected) => topicList.find(topic => topic.id === selected.topic)
```

Comment:
`getSelectedTopic` and `getSelectedEmoji` now both work the same way, as they assume that _both_ associated questions are marked `group_by` in the output JSON